templates/telemetry/clients_daily_v6/view.sql

```diff
@@ -12,19 +12,18 @@ SELECT
     ARRAY(SELECT * FROM UNNEST(environment_settings_intl_requested_locales.list)) AS environment_settings_intl_requested_locales,
     ARRAY(SELECT * FROM UNNEST(environment_settings_intl_system_locales.list)) AS environment_settings_intl_system_locales,
     experiments.key_value AS experiments,
+    SAFE.PARSE_DATE("%F", SUBSTR(profile_creation_date, 0, 10)) AS profile_creation_date,
```

Comment:
This is set up so that `profile_creation_date` will be changed from a `STRING` to a `DATE` in `clients_daily` and `clients_last_seen`, and the old data is converted in views.

I need to verify that this won't cause issues with any scheduled re:dash queries.

For parquet the string format will be restored on export for backwards compatibility.

Reply:
We could instead add a `parsed_profile_creation_date` field alongside the raw value.